src/ui/components/toolbar/Toolbar.tsx

```diff
@@ -55,6 +57,49 @@ const Toolbar = ({
     window.URL.revokeObjectURL(url);
   }, [allEvents]);
 
+  const [highlightActive, setHighlightActive] = React.useState(false);
+  const handleToggleHighlight = () => {
+    setHighlightActive(!highlightActive);
+  };
+
+  React.useEffect(() => {
+    if (highlightActive) {
+      const channel = remoteSubscriber.ns("highlighter");
```

Comment:
This should be moved to highlight context API

Reply:
Fixed